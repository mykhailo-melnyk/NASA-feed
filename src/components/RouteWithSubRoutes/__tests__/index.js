import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import RouteWithSubRoutes from '..';

const testComponent = () => (<div />);

describe('RouteWithSubRoutes component', () => {
  describe('snapshots', () => {
    it('render', () => {
      const props = {
        path: '/',
        exact: true,
        component: testComponent,
      };
      const component = renderer.create(
        <MemoryRouter><RouteWithSubRoutes {...props} /></MemoryRouter>,
      );
      const tree = component.toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
