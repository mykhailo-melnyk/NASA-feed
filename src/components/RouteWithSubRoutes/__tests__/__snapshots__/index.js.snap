// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RouteWithSubRoutes component snapshots render 1`] = `<div />`;
